protocol/messenger_communities.go

```diff
@@ -3785,6 +3785,7 @@ func (m *Messenger) handleSyncInstallationCommunity(messageState *ReceivedMessag
 		logger.Debug("m.handleCommunityDescription error", zap.Error(err))
 		return err
 	}
+	descriptionOutdated := err == communities.ErrInvalidCommunityDescriptionClockOutdated
```

Comment:
Not your code, but how about renaming 
`ErrInvalidCommunityDescriptionClockOutdated` to 
`ErrCommunityDescriptionOutdated`?

Because it's either Invalid or outdated.
Also, `Outdated` already means time, so no reason for explicit `Clock`.

Reply:
I agree with your points, but also there is a convention I tried to follow:
```
var ErrInvalidCommunityDescriptionClockOutdated = errors.New("invalid community description outdated clock")
var ErrInvalidCommunityDescriptionNoOrgPermissions = errors.New("invalid community description no org permissions")
var ErrInvalidCommunityDescriptionNoChatPermissions = errors.New("invalid community description no chat permissions")
var ErrInvalidCommunityDescriptionUnknownChatAccess = errors.New("invalid community description unknown chat access")
var ErrInvalidCommunityDescriptionUnknownOrgAccess = errors.New("invalid community description unknown org access")
var ErrInvalidCommunityDescriptionMemberInChatButNotInOrg = errors.New("invalid community description member in chat but not in org")
var ErrInvalidCommunityDescriptionCategoryNoID = errors.New("invalid community category id")
var ErrInvalidCommunityDescriptionCategoryNoName = errors.New("invalid community category name")
var ErrInvalidCommunityDescriptionChatIdentity = errors.New("invalid community chat name, missing")
var ErrInvalidCommunityDescriptionDuplicatedName = errors.New("invalid community chat name, duplicated")
var ErrInvalidCommunityDescriptionUnknownChatCategory = errors.New("invalid community category in chat")
```

To keep the convention, I tried to change it to `ErrInvalidCommunityDescriptionOutdated`, but it sounds even more odd :sweat_smile:

Reply:
🆗